index.bs

```diff
@@ -12995,207 +12872,301 @@ with the following characteristics:
         The [=map size getter=] is a [=built-in function object=]
         whose behavior when invoked is as follows:
 
-        1.  Let |O| be the <emu-val>this</emu-val> value.
-        1.  If |O| [=is a platform object=],
-            then [=perform a security check=], passing:
-            *   the platform object |O|,
-            *   the identifier "<code>size</code>", and
-            *   the type "<code>getter</code>".
-        1.  If |O| does not [=implement=] <var ignore>A</var>, then [=ECMAScript/throw=] a {{ECMAScript/TypeError}}.
-        1.  Let |map| be the {{ECMAScript/Map}} object that is the value of |O|’s \[[BackingMap]] [=internal slot=].
-        1.  Return [=!=] <a abstract-op>Get</a>(|map|, "<code>size</code>").
+        1.  Let |O| be the <emu-val>this</emu-val> value, [=implementation-checked=] against <var ignore>a</var> with identifier "<code>size</code>" and type "<code>getter</code>".
+        1.  Let |map| be the [=map entries=] of the IDL value
+            that represents a reference to |O|.
+        1.  Return |map|'s [=map/size=], [=converted to an ECMAScript value=].
     </div>
 
     The value of the [=function object=]’s <code class="idl">length</code> property is the Number value <emu-val>0</emu-val>.
 
     The value of the [=function object=]’s <code class="idl">name</code> property is the String value "<code>size</code>".
 
 
-<h5 id="es-map-entries">entries</h5>
+<h5 id="es-map-iterator" oldids="es-iterator, es-iterators">@@iterator</h5>
 
-An <code class="idl">entries</code> data property must exist on
+There must exist a data property whose name is the {{@@iterator}} symbol on
 |A|’s [=interface prototype object=]
 with attributes { \[[Writable]]: <emu-val>true</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val> }
 and whose value is the [=function object=] that is the value of
-the {{@@iterator}} property.
+the <code>entries</code> property.
 
+<div algorithm>
+    To <dfn>create a map iterator</dfn>
+    from a [=/map=] |map|
+    and a |kind| which is either "<code>key+value</code>", "<code>key</code>", or "<code>value</code>"):
+
+    1. Let |closure| be a new [=/Abstract Closure=] with no parameters
+        that captures |map| and |kind| and performs the following steps when called:
+        1. [=map/For each=] |key| → |value| of |map|:
+            1. Set |key| and |value| to each [=converted to an ECMAScript value=].
+            1. If |kind| is "<code>key</code>", let |result| be |key|.
+            1. Else if |kind| is "<code>value</code>", let |result| be |value|.
+            1. Else, let |result| be [$CreateArrayFromList$](« |key|, |value| »).
+            1. Perform [=?=] [$GeneratorYield$]([$CreateIterResultObject$](|result|, <emu-value>false</emu-value>)).
```

Comment:
```suggestion
            1. Perform [=?=] [$GeneratorYield$]([$CreateIterResultObject$](|result|, <emu-val>false</emu-val>)).
```

I found three other instances, so a general Ctrl+F for "emu-value" would be worthwhile.

Reply:
wtf, i also had two `</emu>`